runtime/src/lib.rs

```diff
@@ -211,7 +235,18 @@ fn handle_command(drivers: &mut Drivers) -> CaliptraResult<MboxStatusE> {
         CommandId::FIRMWARE_LOAD => Err(CaliptraError::RUNTIME_UNIMPLEMENTED_COMMAND),
         CommandId::GET_IDEV_CSR => Err(CaliptraError::RUNTIME_UNIMPLEMENTED_COMMAND),
         CommandId::GET_LDEV_CERT => Err(CaliptraError::RUNTIME_UNIMPLEMENTED_COMMAND),
-        CommandId::INVOKE_DPE => Err(CaliptraError::RUNTIME_UNIMPLEMENTED_COMMAND),
+        CommandId::INVOKE_DPE => {
+            let resp = InvokeDpeCmd::execute(drivers, cmd_bytes)?;
+            match resp {
+                MailboxResp::InvokeDpeCommand(InvokeDpeResp {
+                    data_size, data, ..
+                }) => {
+                    drivers.mbox.write_response(&data[..data_size as usize])?;
+                    Ok(resp)
+                }
+                _ => Err(CaliptraError::RUNTIME_INVOKE_DPE_FAILED),
+            }
```

Comment:
Could you make this a function on the `InvokeDpeResp`?

I'm thinking something like

```rust
impl InvokeDpeResp {
  fn write_to_mbox(&self, mbox: &mut Mailbox) -> CaliptraResult<()> {}
}
```

And then we can have a `write_to_mbox` function on MailboxResponse, which calls `write_to_mbox` on the individual response types. That way down on line 233 we can just call `resp.write_to_mbox`.

For all the non-dpe commands, you can just call `Packet::copy_to_mbox(drivers, &mut resp)` for now